src/main/java/uk/gov/justice/hmpps/prison/api/model/Contact.java

```diff
@@ -67,19 +67,19 @@ public class Contact {
     @Schema(description = "id of the person contact", example = "5871791")
     private Long personId;
 
-    @Schema(required = true, description = "Active indicator flag.", example = "true", allowableValues = {"true","false"})
+    @Schema(required = true, description = "Active indicator flag.", example = "true")
```

Comment:
Suggestion.  We could move the allowable values into the description if this is a concern?  You can see the allowable values thought if you look at the enum.